docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -1095,9 +1098,12 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "old key [0 0]\n",
-      "    \\---SPLIT --> new key    [4146024105  967050713]\n",
-      "             \\--> new subkey [2718843009 1272950319] --> normal [-1.2515389]\n"
+      "old key Array((), dtype=key<fry>) overlaying:\n",
+      "[0 0]\n",
+      "    \\---SPLIT --> new key    Array((), dtype=key<fry>) overlaying:\n",
+      "[4146024105  967050713]\n",
+      "             \\--> new subkey Array((), dtype=key<fry>) overlaying:\n",
+      "[2718843009 1272950319] --> normal [-1.2515389]\n"
```

Comment:
Unrelated to the change here, but this output formatting is not great given the new repr of key arrays.

Reply:
Good point! I've actually updated the `ipynb` file to minimize the diff here but I'm happy to take a look at this point separately.